scripts/archive.coffee

```diff
@@ -54,6 +56,9 @@ module.exports = (robot) ->
           pr_body += "[`archive` command of our chatbot](https://github.com/patcon/hyphacoop-chatbot/tree/master/README.md#archive),"
           pr_body += " kicked off by user `#{user}` in `#{room}`."
 
+          rendered_url = "https://#{config.archive_repo_login}.github.io/#{config.archive_repo_name}/#{slug}.html"
```

Comment:
I'm into this, just curious on variable name choice "`archive_repo_login`"

Reply:
heh :) github "users" are called "login" in the GitHub API, so as not to imply anything specific to organization or individual username. so it leaked into my naming, but is this better?

```suggestion
          rendered_url = "https://#{config.archive_user_name}.github.io/#{config.archive_repo_name}/#{slug}.html"
```